.base_api

```diff
@@ -1 +1 @@
-0e82bd13eb3d3da19f32a65a61defc00f4fe8c10
\ No newline at end of file
+7b8d4d56aae4d0b300f3583776bfe15040e802b6
```

Comment:
What exactly was changed in https://github.com/mapbox/mapbox-navigation-ios/pull/4277 that requires this API bump? It looks like unexpected change to me, as only new API was added, no (I might be missing something in your previous PR though)?

Reply:
As in https://github.com/mapbox/mapbox-navigation-ios/pull/4277#issuecomment-1339912700 , previously, only the `NavigationViewControllerDelegate` has the method as the `NavigationMapView`'s delegate to provide the layer for restricted route layer as in `navigationViewController(_:routeRestrictedAreasLineLayerWithIdentifier:sourceIdentifier:)`. #4277 also add this one to the `CarPlayManagerDelegate`, `CarPlayMapViewControllerDelegate` and `CarPlayNavigationViewControllerDelegate`. Because CarPlay already have the methods for route layer and casing layer already, but the restricted layer one is missing